src/Slider/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useRef } from "react";
+import { useNumberFormatter, useSlider, VisuallyHidden } from "react-aria";
+import { useSliderState } from 'react-stately';
+import Thumbs from "./Thumbs";
+
+interface Props {
+  label: string;
+  output?: string;
+  higherName?: string;
+  lowerName?: string;
+  formatOptions: Intl.NumberFormatOptions;
+  maxValue: number;
+  defaultValue?: number[];
+  step: number;
+  value?: number[];
+  setValue?: CallableFunction
```

Comment:
Do we need a `onChange` prop here?

```
<Slider
   value={valueStateVar}
   onChange={({ min, max }) => setValueStateVar([min,max])}
   ✂️ 
```

Reply:
yep you're right - changed this to `onChange`.